docs/indexing.md

```diff
@@ -197,7 +197,15 @@ above to do this)
 
 ### Swap in the new index
 
-First, update the index managers to have the new solr_collection values.
+Before you swap in the new index, make sure that the solr cloud is all working as expected:
+
+1. Open the solr console
+2. Open the Cloud sub-menu
```

Comment:
In Markdown this should just be `1.` for all the numbers and then markdown will take care of the numbering.